src/js/service/collectElementService.js

```diff
@@ -72,6 +72,9 @@ collectElementService.initWithGrid = function (gridData, dontAutoPredict) {
                 }, null);
             collectMode = copy.mode || collectMode;
             convertToLowercaseIfKeyboard = copy.convertToLowercase !== false;
+            // Defaults for new/legacy grids that may not have these flags persisted
+            copy.displayUpsideDown = copy.displayUpsideDown !== undefined ? copy.displayUpsideDown : false;
+            copy.rotationToggleActive = false;
```

Comment:
just one line: copy.rotationActive = !!copy.displayUpsideDown;
just one local variable to determine if toggle or not - with initial value of the element property.